### PR TITLE
Update language update popup

### PR DESCRIPTION
**Pull request by onein528**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #issue...
- Related #issue...

**Details of Changes**
Add details of changes here.
- Added corner radius property
- Update "yes" button style

**Validation**
How did you test these changes?
- [ ] Built and ran the app
- [ ] Tested the changes for accessibility

**Screenshots (optional)**
Before:
image

### CODE REVIEW

**Comment by onein528** [comment]
Unfortunately I can't run the project right now.
So the location to set the radius of the corner may be wrong.

File: src/Files/UserControls/RestartControl.xaml [hunk 1]
```diff
@@ -35,7 +36,7 @@
                         <Button
                             x:Name="YesButton"
                             x:Uid="ButtonYes"
-                            Background="{StaticResource SystemAccentColor}"
+                            Style="{ThemeResource AccentButtonStyle}"
```
Comment (yaichenbaum) [inline src/Files/UserControls/RestartControl.xaml:39]: Is there any reason to change the full style?

**Comment by yaichenbaum** [comment]
@onein528 can you resolve the merge conflicts?

File: src/Files/UserControls/RestartControl.xaml (hunk 1 above)
Comment (onein528) [reply src/Files/UserControls/RestartControl.xaml:40]: This is because the hover style does not apply.

**Comment by onein528** [comment]
Did I resolve the conflict?

File: src/Files/UserControls/RestartControl.xaml (hunk 1 above)
Comment (onein528) [reply src/Files/UserControls/RestartControl.xaml:40]: If you don't need it, you can delete it.

**Comment by yaichenbaum** [comment]
@onein528 this doesn't seem to have fixed the corner radius.

**Comment by onein528** [comment]
Ummm,,, I have no other ideas...
[Preferences.xaml L336](https://github.com/files-community/Files/blob/main/src/Files/Views/SettingsPages/Preferences.xaml#L336)
```diff
 <UserControls:RestartControl
     x:Name="RestartControl"
     VerticalAlignment="Bottom"
-    ShowDialog="{Binding ShowRestartControl, Mode=TwoWay}" />
+    ShowDialog="{Binding ShowRestartControl, Mode=TwoWay}"
+    CornerRadius="{StaticResource OverlayCornerRadius}"/>
```

**Comment by yaichenbaum** [comment]
> Ummm,,, I have no other ideas... [Preferences.xaml L336](https://github.com/files-community/Files/blob/main/src/Files/Views/SettingsPages/Preferences.xaml#L336)
> 
> ```diff
>  <UserControls:RestartControl
>      x:Name="RestartControl"
>      VerticalAlignment="Bottom"
> -    ShowDialog="{Binding ShowRestartControl, Mode=TwoWay}" />
> +    ShowDialog="{Binding ShowRestartControl, Mode=TwoWay}"
> +    CornerRadius="{StaticResource OverlayCornerRadius}"/>
> ```

You'll probably have to play around with the template of the control.

**Comment by onein528** [comment]
come back later

**Pull request closed without merging**